### ISSUE TITLE
Metadata ID Format

### ISSUE THREAD

**Issue by mej54** [walaj/svaba#24]
Hi, 

I've noticed that the paths to the bam files I'm using in my svaba runs are used as the sample IDs in the vcf files. I'm wondering if this is the intended output, or if there will be an option to specify each sample ID for vcf format purposes? Having flags such as --tumor-sample-name and --normal-sample-name would be really helpful so that we don't run into any format issues due to the backslashes in the bam file paths.

**Comment by ahwanpandey** [comment]
Yes I wanted to do the same as well. Are there any plans to implement this?

Thanks.
